spec/jobs/facilities/pssg_download_spec.rb

```diff
@@ -89,9 +99,10 @@
     end
 
     it 'leaves facility with original drive time band' do
+      create :vha_402QA
```

Comment:
I found that because the facility wasn't being created drivetime bands were not actually being inserted for it. I changed this to a facility that already has a factory.

Reply:
Silent failure there could be difficult to debug in the future. Should we add a log?

Reply:
There actually is logging for this path through the logic. I just felt like not actually changing any drivetime bands wasn't fulling the promise of this test.
https://github.com/department-of-veterans-affairs/vets-api/blob/5d9af96ab8f37d4e01c9264a13e0e695200100bc/app/workers/facilities/pssg_download.rb#L36